### PR TITLE
fix: prevent js errors in case of using nette.ajax while no grid is pres...

### PR DESCRIPTION
**Pull request by LeonardoCA**
...ent on recent page

### CODE REVIEW

File: js/nextras.datagrid.js [hunk 1]
```diff
@@ -39,10 +39,14 @@ $.nette.ext('datagrid', {
 		});
 	},
 	before: function(xhr, settings) {
-		this.grid = settings.nette.el.parents('.grid');
+		if ((typeof settings.nette !== typeof undefined) && (settings.nette !== false)) {
```
Comment (hrach) [inline js/nextras.datagrid.js]: neni to typeof u undefined zbytecne? a settins.nette muze byt false? :) jinak super!

Comment (vojtech-dobes) [inline js/nextras.datagrid.js]: Myslím že `settings.nette` nikdy není `false` :).

Comment (LeonardoCA) [inline js/nextras.datagrid.js]: i am not sure about best solution, I just used what's mentioned here https://stackoverflow.com/questions/1318076/jquery-hasattr-checking-to-see-if-there-is-an-attribute-on-an-element

**Comment by hrach** [comment]
Thank you!

**Pull request merged**